ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -66,7 +66,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(RangerRestClientImpl.class);
   private static final String RANGER_REST_URL_EXPORTJSONFILE = "/service/plugins/policies/exportJson";
   private static final String RANGER_REST_URL_IMPORTJSONFILE =
-      "/service/plugins/policies/importPoliciesFromFile?updateIfExists=true";
```

Comment:
Why we should remove the slash here and not also above (i.e., `RANGER_REST_URL_EXPORTJSONFILE`).

Reply:
There was a different condition for the RANGER_REST_URL_EXPORTJSONFILE. I have unified it and added tests as well.